packages/content/src/requests/items/formField/settings/RatingFormFieldSettings.ts

```diff
@@ -0,0 +1,29 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { FormFieldSettings, FormFieldSettingsJSON, IFormFieldSettings } from "./FormFieldSettings";
+
+export const minRating = 1;
```

Comment:
This is a very bad naming for a field that is exported from the whole library.

Imagine you install this lib from npmjs and find `minRating` exported without any further explanation...

Please make this a static field (or maybe better would be a getter) of RatingFormFieldSettings OR (and I would be a fan of this approach) just remove it. It's well known that 0 is not a valid rating and we can just assume the scale.